### PR TITLE
[WIP] move to Scala 2.13.0-RC1 (drop -M5)

### PR DESCRIPTION
**Pull request by SethTisue**
this is just a sketch to get the ball rolling, I don't intend
to finish it myself

not even sure if it's to the right branch, not sure what the plan is around ScalaCheck 1.14.0 being the only ScalaCheck available for 2.13 (for now? not sure if anyone has tried to push for 1.13.x release on 2.13)

### CODE REVIEW

**Comment by SethTisue** [comment]
you may need to revisit https://github.com/typelevel/cats/issues/2360

File: core/src/main/scala/cats/ApplicativeError.scala [hunk 1]
```diff
@@ -82,7 +82,7 @@ trait ApplicativeError[F[_], E] extends Applicative[F] {
    * `F[A]` values.
    */
   def recover[A](fa: F[A])(pf: PartialFunction[E, A]): F[A] =
-    handleErrorWith(fa)(e => (pf.andThen(pure)).applyOrElse(e, raiseError))
+    handleErrorWith(fa)(e => (pf.andThen(pure _)).applyOrElse(e, raiseError _))
```
Comment (jozic) [inline core/src/main/scala/cats/ApplicativeError.scala]: may i ask why those underscores are needed now? What have changed to make them required?

Comment (SethTisue) [reply core/src/main/scala/cats/ApplicativeError.scala]: I think it's https://github.com/scala/scala/pull/7263

**Comment by SethTisue** [comment]
superseded by #2792

**Pull request closed without merging**